### DISCUSSION TITLE
Let share if you have success App on store

### DISCUSSION THREAD

**Discussion by minhcasi** [inspireui/support#5106] [Working on Features]
Please share your live app on the app stores to get more ratings from another user, or any suggestion ideas.

**Comment by sirlojik** [comment]
![Duka Online Shopping](https://www.duka.ug/wp-content/uploads/2021/04/featured-image.png)
Hi, i run Duka and today i released fluxstore variant to playstore 
https://play.google.com/store/apps/details?id=com.duka.app
Screenshot 1 | Screenshot 2 | Screenshot 3
------------ | ------------- | -------------
![](https://www.duka.ug/wp-content/uploads/2021/04/Google-Pixel-4-XL.png) | ![](https://www.duka.ug/wp-content/uploads/2021/04/Google-Pixel-4-XL-1.png) | ![](https://www.duka.ug/wp-content/uploads/2021/04/Google-Pixel-4-XL-2.png)

Screenshot 4 | Screenshot 5 | Screenshot 6
------------ | ------------- | -------------
![](https://www.duka.ug/wp-content/uploads/2021/04/Google-Pixel-4-XL-3.png) | ![](https://www.duka.ug/wp-content/uploads/2021/04/Google-Pixel-4-XL-4.png) | ![](https://www.duka.ug/wp-content/uploads/2021/04/Google-Pixel-4-XL-5.png)

the vendor app is also on playstore [Duka Vendor App](https://play.google.com/store/apps/details?id=com.dukavendor.app). Visit[ https://www.duka.ug](https://www.duka.ug) for more.

MY feature requests to have everything 100% would be
1) Assigning delivery agents within store manager app.
2) Delivery app, although i have settled for one i found on code canyon. " Delivery Person app".
3) Support Terawallet
4) Support [woo sell services](https://wbcomdesigns.com/downloads/woo-sell-services/) (This is trendy now like fiver). It already supports wcfm
5) Support [woocommerce deposit](https://woocommerce.com/products/WooCommerce-deposits/) plugin... to allow people to pay in installments esp when used with woo sell services. This plugin also supports wcfm

**Comment by sergeyshaklein** [comment]
Hi.
Green store )

App Store: 
https://apps.apple.com/us/app/свежо/id1545362429

google play
https://play.google.com/store/apps/details?id=com.coffye.xneaaedkicoetl

**Comment by QuickBasket** [comment]
Am not a techy myself , but, with the help of InspireUI support team my woo commerce based app Quick Basket is live on both stores and am getting personal messages about it being nice! Thanks InspireUI team for all your help! 

https://onelink.to/Quickbasket

**Reply by Yan-Aleman** [reply]
> Am not a techy myself , but, with the help of InspireUI support team my woo commerce based app Quick Basket is live on both stores and am getting personal messages about it being nice! Thanks InspireUI team for all your help!
> 
> https://onelink.to/Quickbasket

Yes, they are really the best.  InspireUI 🔥 🔥 🔥

**Comment by trunghieuvn** [comment]
![inspireui-fluxstore](https://user-images.githubusercontent.com/19247680/141238003-7673625e-31f0-4844-80d1-6b1d91abb00f.png)

**Comment by parvej4u2** [comment]
## Promotional Video Ads

<p align="center"> 
<a href="https://www.youtube.com/watch?v=YTjLEd2MilY&ab_channel=JustGooIN" rel="JustGoo App"><img src="https://food.justgoo.in/wp-content/uploads/2021/11/justgoo_succesful00.webp" alt="JustGoo App Promo" /></a>
</p>

## App Screenshot
<table align="center">
  <tr>
    <th>Screenshot 1</th>
    <th>Screenshot 2</th>
    <th>Screenshot 3</th>
     <th>Screenshot 4</th>
    <th>Screenshot 5</th>
    <th>Screenshot 6</th>
  </tr>
  <tr>
    <td><img src="https://play-lh.googleusercontent.com/u87rETdIAsDXKpvFEdx4bM3Wv8FLK8zuDAI72WnMKiTEUtRVLypV4hVHf88gwi8sVBU=w720-h310-rw"></td>
    <td><img src="https://play-lh.googleusercontent.com/CkIkjNIJlF2_ntGvtogWcIzArZNr3Q5w8ZVacLKPQKfcOPHLXii-j-5ZE5SL3VpBZw=w720-h310-rw"></td>
    <td><img src="https://play-lh.googleusercontent.com/VFIO-17j0V5A8TXZEVlZHnAZ2Xrzjg6e0JjTBTGa8N-Cz1_hQfS9S092Zay6w-SqfdEu=w720-h310-rw"></td>
    <td><img src="https://play-lh.googleusercontent.com/dLblcvxa5iwhMFeK7B5wf_1fgNKPzmFRNyGo9R-3smNymeVOCLrAktkg10R6m95HZZo=w720-h310-rw"></td>
    <td><img src="https://play-lh.googleusercontent.com/trl_MlWvnBq2c_Fq8hJzn9kfHpPQQTk_aPsbAwGL5XPYMBO6jXbTATLSFz6wQnYpCfc=w720-h310-rw"></td>
    <td><img src="https://play-lh.googleusercontent.com/UiFq1y1_h-z1rwQLJXbF400HqlhEBNGj0UWVDpgUscXVdJ7SL5UOlOvGYVUo3cgRB5Y=w720-h310-rw"></td>   
  </tr>
</table>

## App Features

 - [x] Hot Deals & Offers: Get access to everyday deals, festive discounts and exclusive app-only offers.  
  - [ ] Wishlist: Add products to your "Wishlist" to view or purchase it later.  
 - [ ] Search Simplified: Browse or Search for your favorite products by categories and brands.  
  - [ ] Ratings & Reviews: Make a right buying decision with the help of "Reviews and Ratings" on product.  
  - [ ] Secure Payment: Multiple payment options - Cash on Delivery, UPI, Credit/Debit Card, Net Banking  
  - [ ] Sharing is Easier: Share products you like with your Friends & Family.  
  - [ ] Instant Notifications: Be the first to know the best deals, offers & coupons available.  
  - [ ] My Account: View order details, track orders, dark theme, notification control.

## App Ratings
<p align="center"> 
<img src="https://food.justgoo.in/wp-content/uploads/2021/11/ratings.jpg">
</p>


## App Download Link

<p align="center"> 
<img src="">
<a href="https://play.google.com/store/apps/details?id=com.justgoo.android" rel="JustGoo App"><img src="https://food.justgoo.in/wp-content/uploads/2021/08/play-store-icon1.png" alt="JustGoo App Download" /></a>
</p>
